src/main/java/domain/Number.java

```diff
@@ -0,0 +1,21 @@
+package domain;
+
+public class Number {
+    private int value;
+
+    public Number(String value) {
+        try {
+            this.value = Integer.parseInt(value);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException("Invalid number: " + value);
+        }
+    }
+
+    public Number(int value) {
+        this.value = value;
+    }
+
+    public int copyValue() {
```

Comment:
```suggestion
    public int value() {
```
개인적인 의견이지만 copy보다는 값 그대로 반환한다는 의미에서 `value()`가 좀 더 자연스럽지 않을까요?

Reply:
음.. 확실히 복사라고하기에는 primitive 타입에 붙이기는 부자연스러운부분이 있네요. 수정해두겠습니다.